scripts/boston_cityhall_electricity.json

```diff
@@ -0,0 +1,35 @@
+{
+    "citation": "FILL",
+    "description": "FILL",
+    "homepage": "FILL",
+    "keywords": [],
+    "licenses": [
+        {
+            "name": "FILL"
+        }
+    ],
+    "name": "FILL",
+    "resources": [
+        {
+            "dialect": {},
+            "name": "tmp7mv7ycav",
```

Comment:
This `name` is that table name, we can give our table a better name. For example we could say `consumption` or `usage`, etc

Reply:
Also add "path":"tmp7mv7ycav.csv" in the resources. You can put that below 'name'